src/SearchResults.js

```diff
@@ -0,0 +1,36 @@
+import moment from "moment";
```

Comment:
You forgot to import `React` in this component, so it failed to load on the page.